src/agri_ai/domain/master_data_matcher.py

```
@@ -0,0 +1,378 @@
+"""
+マスターデータマッチャー
+
+抽出された情報とマスターデータの照合・検証を専門的に処理
+I/Oに依存しない純粋なドメインロジック
+"""
+
+import logging
+from typing import Dict, Any, List, Optional, Tuple
```

Ruff (0.12.2)

`typing.Optional` imported but unused

Remove unused import

(F401)